spec/factories/card.rb

```diff
@@ -3,5 +3,6 @@
     original_text "How are you?"
     translated_text "Kamusta?"
     review_date DateTime.now.getutc
+    user_id 1
```

Comment:
@davetoxa wow.. i'm surprised it works. how did that work? the others are obvious (method call + an argument) but i'm confused how passing an object is magically understood by Rails.

what does each line of code within the block return here?